From: weiping zhang <zhangweiping@didichuxing.com>
Date: Fri, 18 Aug 2017 14:49:30 +0800
Subject: [PATCH] scsi: sd: remove duplicated setting of gd->minors
Git-commit: 5f800c87dadf0857b31ccb6217d6d42d4c1fe157
Patch-mainline: v4.14-rc1
References: bsc#1118139

gd->minors has been set when call alloc_disk() in sd_probe.

Signed-off-by: weiping zhang <zhangweiping@didichuxing.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 drivers/scsi/sd.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/scsi/sd.c b/drivers/scsi/sd.c
index 7c0f9eb5a5fd..6549e5ce09ca 100644
--- a/drivers/scsi/sd.c
+++ b/drivers/scsi/sd.c
@@ -3223,7 +3223,6 @@ static void sd_probe_async(void *data, async_cookie_t cookie)
 
 	gd->major = sd_major((index & 0xf0) >> 4);
 	gd->first_minor = ((index & 0xf) << 4) | (index & 0xfff00);
-	gd->minors = SD_MINORS;
 
 	gd->fops = &sd_fops;
 	gd->private_data = &sdkp->driver;
-- 
2.12.3

